rpm-ostree-container-bootc.ks.in

```diff
@@ -3,7 +3,10 @@
 # depends on the referenced ostree container being bootable
 
 # Use the default settings.
-%ksappend common/common_no_payload.ks
+%ksappend common/common_no_storage_and_payload.ks
+%ksappend payload/default_packages.ks
```

Comment:
Including `%packages` section contradicts with a container image payload, at least for now (even though it may just be ignored by Anaconda).

Reply:
Ah, good catch, thank you!

Reply:
Updated

Reply:
ah, that would be great

Reply:
You are right, thank you, updated.